bindings/python/strip_header.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generate a CFFI.cdef() string from a C header file
+
+Usage: python strip_header.py ../../src/mongocrypt.h | pbcopy
```

Comment:
Same note about this being OS-X only. 

Suggest: 
```
Usage (on OS-X): python strip_header.py ../../src/mongocrypt.h | pbcopy
```

Reply:
Done.